tests/integration_tests/data_steward/tools/top_heel_errors_test.py

```diff
@@ -87,11 +86,10 @@ def load_test_data(self, hpo_id: str = None):
         test_file_path: str = os.path.join(test_util.TEST_DATA_PATH,
                                            test_file_name)
 
-        target_bucket = self.storage_client.get_bucket(self.bucket)
-        test_blob = target_bucket.blob(test_file_name)
+        test_blob = self.drc_bucket.blob(test_file_name)
         test_blob.upload_from_filename(test_file_path)
 
-        gcs_path: str = f'gs://{self.bucket}/{test_file_name}'
+        gcs_path: str = f'gs://{self.drc_bucket.name}/{test_file_name}'
```

Comment:
```
     test_blob = self.drc_bucket.blob(f'{table_id}.csv')
     test_file_path: str = os.path.join(test_util.TEST_DATA_PATH,
                                           test_blob.name)
     test_blob.upload_from_filename(test_file_path)
     gcs_path: str = f'gs://{self.drc_bucket.name}/{test_blob.name}'
```

Reply:
I have a suggestion. You could remove and replace test_file_name to use the name attribute from test_blob.

Reply:
Ah,  I think I see what you're doing here.  Create the blob with the string once then use the name attribute?

Reply:
Yes